stripper/ze_atix_panic_v1.cfg

```diff
@@ -162,4 +142,127 @@ add:
 	"speed" "320"
 	"StartDisabled" "0"
 	"alternateticksfix" "0"
+}
+
+;Exploding prop like other atix.
+add:
+{
+	"classname" "logic_timer"
+	"targetname" "PreventDelay2"
+	"spawnflags" "0"
+	"StartDisabled" "1"
+	"UseRandomTime" "1"
+	"LowerRandomBound" "7"
+	"UpperRandomBound" "13"
+	"OnTimer" "prop_phys*RunScriptCodeforeach(a1,_ in {explosion_basic=0}) a <- a101"
+	"OnTimer" "prop_phys*RunScriptCodeb <- 'b'.tochar() + 'r'.tochar() + 'e'.tochar() + 'a'.tochar() + 'k'.tochar()01"
+	"OnTimer" "prop_phys*RunScriptCodefunction c(){ DispatchParticleEffect(a, self.GetOrigin(), Vector(null));EntFireByHandle(self, b, b, 0.02, null, null); }01"
+	"OnTimer" "prop_phys*RunScriptCodeif (RandomInt(0, 7) == 1) c();0.01-1"
```

Comment:
You should only target prop_physics_override with this. Because unlike b3t_p, v1 has a prop_physics_multiplayer "radio" that should not be getting killed by this anti-delay mechanism.